### PR TITLE
add arrowParens to prettier config

### PR DESCRIPTION
**Pull request by SaraVieira**
Also adds a max height to make it scrollable

### CODE REVIEW

File: packages/app/src/app/components/CodeEditor/Configuration/elements.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ export const Container = styled.div`
   padding: 1rem;
   color: rgba(255, 255, 255, 0.8);
   box-sizing: border-box;
-
-  overflow: auto;
+  max-height: 100%;
+  background: red;
```
Comment (CompuIves) [inline packages/app/src/app/components/CodeEditor/Configuration/elements.ts]: 😅?

File: packages/common/src/templates/configuration/prettierRC/ui.tsx [hunk 2]
```diff
@@ -152,6 +152,19 @@ export class ConfigWizard extends React.Component<ConfigurationUIProps> {
             line instead of being alone on the next line.
           </ConfigDescription>
         </PaddedConfig>
+        <PaddedConfig>
```
Comment (CompuIves) [inline packages/common/src/templates/configuration/prettierRC/ui.tsx]: Nice! I think we also need to add this to the preferences: `packages/app/src/app/pages/common/Modals/PreferencesModal/CodeFormatting/Prettier/index.js`

File: packages/app/src/app/components/CodeEditor/Configuration/elements.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/app/components/CodeEditor/Configuration/elements.ts]: LOL! Removed, it was the one that didn't work ahah

File: packages/common/src/templates/configuration/prettierRC/ui.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/common/src/templates/configuration/prettierRC/ui.tsx]: Added there too :)

**Comment by CompuIves** [comment]
Great! Looks good!

**Pull request merged**